analysis_options.yaml

```diff
@@ -50,7 +50,7 @@ linter:
     # http://dart-lang.github.io/linter/lints/always_specify_types.html
     # recommendation: optional
     # 0 issues
-    - always_specify_types
+#    - always_specify_types
```

Comment:
This was commented out as it was causing an analyzer warning as this conflicts with two ohter rules (`avoid_types_on_closure_parameters` and `omit_local_variable_types`). [Looks like most repos leave this in their analysis_options.yaml file, but comment it out like this.](https://sourcegraph.wk-dev.wdesk.org/search?q=always_specify_types+lang:yaml&patternType=literal)

For that reason I opted to leave it in here, but commented it out.